feature-group-definitions/structured-clone.yml

```
@@ -1,4 +1,5 @@
 name: structuredClone()
+description: The global `structuredClone()` JavaScript method creates a deep clone of an object by using the structured clone algorithm. The method also allows for transferring values from the original object to the cloned object.
```

Explaining tranferables is a bit tricky. It's not an optional use case you might have, it's something you have to do because `structuredClone()` will throw if you don't. Trying:

```suggestion
description: The `structuredClone()` global method creates a deep copy of an object. Values that cannot be cloned can instead be transferred, making the original value no longer usable.
```

I dropped the algorithm reference because it's a spec concept that most devs won't know. If we could link it that might be useful, but we can't now.